### ISSUE TITLE
Show empty message

### ISSUE THREAD

**Issue by bjarnef** [leekelleher/umbraco-contentment#103]
Based on Contentment v1.4.1 I noticed when no results are found just the searchbox is shown.

![image](https://user-images.githubusercontent.com/2919859/116237552-a7de6100-a760-11eb-8072-3bb230c62c7f.png)

It could be nice to show an empty message here as I think that's the approach in Umbraco core using `<umb-empty-state>`, e.g.
https://github.com/umbraco/Umbraco-CMS/blob/9d5a5d886e6b58c4aeda6917ce49ff25d815fe0e/src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/iconpicker/iconpicker.html#L54-L58

**Comment by leekelleher** [comment]
@bjarnef Thanks for the suggestion.

In the Umbraco icon picker example, I was curious how `filtered` was set...

<https://github.com/umbraco/Umbraco-CMS/blob/9d5a5d886e6b58c4aeda6917ce49ff25d815fe0e/src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/iconpicker/iconpicker.html#L45>

...didn't know it could be set inline within an `ng-repeat`. Learned something new today.

**Comment by bjarnef** [comment]
@leekelleher yes, the filtered result can be set as a new scope property. With a large data set it will probably filter the results server side and return a `PagedResult<T>`, so the angular items already are filtered.

**Comment by leekelleher** [comment]
Reading up about `ng-repeat`, there's an `as alias` syntax that can be used to store at as temporary variable within the scope.
<https://docs.angularjs.org/api/ng/directive/ngRepeat#ngRepeat-arguments>

I'll have a play, it'll be good to display a "no results" message. 👍
